### PR TITLE
Library: Port 'CSS Gradients' to Python

### PR DESCRIPTION
**Pull request by UrtsiSantsi**
Marking as draft, since it doesn't work for me - see #713

### CODE REVIEW

**Comment by theCapypara** [comment]
I'm not sure I understand, your commit passed CI. Black expects a new line at the end and your file has one.

File: src/Library/demos/CSS Gradients/main.py [hunk 1]
```diff
@@ -0,0 +1,110 @@
...
+)
+spin_row_angle = workbench.builder.get_object("spin_row_angle")
+button_color_1 = workbench.builder.get_object("button_color_1")
+button_color_2 = workbench.builder.get_object("button_color_2")
+button_color_3 = workbench.builder.get_object("button_color_3")
+gtksource_buffer = workbench.builder.get_object("gtksource_buffer")
+button_copy_css = workbench.builder.get_object("button_copy_css")
+
+
+def generateCss():
```
Comment (theCapypara) [inline src/Library/demos/CSS Gradients/main.py:22]: Please use snake case names (`generate_css`)

File: src/Library/demos/CSS Gradients/main.py [hunk 2]
```diff
@@ -0,0 +1,110 @@
...
+    angle_string = spin_row_angle.get_value()
+    first_color_string = button_color_1.get_rgba().to_string()
+    second_color_string = button_color_2.get_rgba().to_string()
+    third_color_string = button_color_3.get_rgba().to_string()
+
+    css = ""
+    if combo_row_gradient_type.get_selected() == 0:
+        css = """
```
Comment (theCapypara) [inline src/Library/demos/CSS Gradients/main.py:30]: This (and the others) are not valid format strings. You want something like:
```py
f""""
{angle_string}deg
""""
```

**Review by theCapypara — CHANGES_REQUESTED** [review]
There really seems to be something wrong with importing SourceView, I can reproduce this in your example. It might be that we need to make sure GtkSource is loaded in the previewer too, since it's defined in the Builder XML, I did not check that yet. Will have a look.

**Comment by gregorni** [comment]
@UrtsiSantsi Is there any way I can contact you outside GitHub? We're both working on Python demos right now, in parallel, so I thought it would be smart if we talked together to make sure we don't accidentally port the same demo.

If you're on Matrix, you can find me under https://matrix.to/#/@gregorni:gnome.org

**Comment by theCapypara** [comment]
I'll add the Adw and other initialization code to the previewer as well, but please also keep it in the demo, that's good to have for people that try to copy the demos into their own apps.

**Review by theCapypara — APPROVED** [review]

**Pull request merged**